launcher-service/src/controllers/authcontroller.ts

```diff
@@ -120,13 +122,17 @@ export const login = async (
 
     // Generate a new JWT token with the user ID and role
 
-    const token = jwt.sign(
-      { id: user?.id, role: role },
-      process.env.JWT_SECRET,
-      {
-        expiresIn: '5m', // Expires in 5 min
-      }
-    );
+    const token = isAdmin
+      ? jwt.sign(
+          { id: user?.id, role: role, isAdmin: isAdmin },
+          process.env.JWT_SECRET,
+          {
+            expiresIn: '5m', // Expires in 5 min
+          }
+        )
+      : jwt.sign({ id: user?.id, role: role }, process.env.JWT_SECRET, {
+          expiresIn: '5m', // Expires in 5 min
+        });
```

Comment:
Simplify this code like below. 
```
let  payload = {
  id: user.id,
  role
};

if (isAdmin) payload = {...payload, isAdmin}

const token  = jwt.sign(payload, process.env.JWT_SECRET, {
    expiresIn: '5m',
 });

```

Reply:
thank you. that simplifies it much better.